pkg/ccl/cliccl/load.go

```diff
@@ -122,6 +135,55 @@ func runLoadShowSummary(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func runLoadShowIncremental(cmd *cobra.Command, args []string) error {
+
+	path := args[0]
+	if !strings.Contains(path, "://") {
```

Comment:
Is this converting a path like `/path/to/my/file` to `nodelocal://0/path/to/my/file`? If so, I think we might want to support paths to files still with an explicit scheme so the argument stays a URI. Something like: `file://path/to/my/file` perhaps? This change may be large enough that it warrants it's own PR.

Reply:
Agree. Will follow up in another PR.

Reply:
Do we need to keep this if statement around until we add support for that? It doesn't look like we have any tests for `./cockroach load show incremental /path/to/backup`?